keepkey/local/baremetal/fsm.c

```diff
@@ -996,46 +996,46 @@ void fsm_msgGetAddress(GetAddress *msg)
 
 void fsm_msgEthereumGetAddress(EthereumGetAddress *msg)
 {
-	RESP_INIT(EthereumAddress);
+    RESP_INIT(EthereumAddress);
 
-        if (!storage_is_initialized()) {
-		fsm_sendFailure(FailureType_Failure_NotInitialized, "Device not initialized");
-		return;
-	}
+    if (!storage_is_initialized()) {
+        fsm_sendFailure(FailureType_Failure_NotInitialized, "Device not initialized");
+        return;
+    }
 
-        if(!pin_protect("Enter Current PIN"))
-        {
-                go_home();
-		return;
-	}
+    if (!pin_protect_cached()) {
```

Comment:
This is the line that changed. The rest of the changes in this file are formatting difference. Sorry about that.